lib/cpp/test/qt/CMakeLists.txt

```diff
@@ -18,11 +18,15 @@
 #
 
 set(CMAKE_AUTOMOC ON)
-find_package(Qt5 REQUIRED COMPONENTS Test)
+find_package(Qt5 REQUIRED COMPONENTS Test Network)
 set(TQTcpServerTest_Qt5_SOURCES
     TQTcpServerTest.cpp
 )
 add_executable(TQTcpServerTest_Qt5 ${TQTcpServerTest_Qt5_SOURCES})
-target_link_libraries(TQTcpServerTest_Qt5 testgencpp_cob thriftqt5 thrift Qt5::Test)
+target_link_libraries(TQTcpServerTest_Qt5 testgencpp_cob)
+LINK_AGAINST_THRIFT_LIBRARY(TQTcpServerTest_Qt5 thriftqt5)
+LINK_AGAINST_THRIFT_LIBRARY(TQTcpServerTest_Qt5 thrift)
+qt5_use_modules(TQTcpServerTest_Qt5 Test Network)
```

Comment:
According to [Qt5 documentation](http://doc.qt.io/qt-5/cmake-manual.html), *qt5_use_modules* is the way to go with CMake **prior to 2.8.11**.
Since root CMakeLists.txt requires 2.8.12, I believe imported targets (*Qt5::Test Qt5::Network*) are more appropriate than the macro here.